de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/ui/views/SarosView.java

```diff
@@ -441,6 +446,13 @@ public void menuAboutToShow(final IMenuManager manager) {
              * version 14.1.31)
              */
             // manager.add(getAction(SkypeAction.class));
+
+            // TODO: Replace Server check once other implementations support JoinSessionRequests
+            if (contacts.size() == 1
+                && discoveryManager.isFeatureSupported(
```

Comment:
It would be better to query `discoveryManager` separately beforehand and do a null-check on the result before checking the boolean value.

As the method `isFeatureSupported` returns a `Boolean` object (and the primitive type `boolean`), the returned object can be null, at which point the implicit unboxing in the if statement will throw a nullpointer exception.